src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -920,14 +920,20 @@ def __call__(self, result):
             result = super(DeploymentOutputLongRunningOperation, self).__call__(result)
             outputs = None
             try:
-                outputs = result.properties.outputs
+                if isinstance(result, str) and result:
+                    try:
+                        obj = json.loads(result)
+                        return obj
+                    except json.decoder.JSONDecodeError:
```

Comment:
What if the error is not json.decoder.JSONDecodeError?

Reply:
It will be throw out.